backend/tasks.py

```diff
@@ -21,6 +21,39 @@
     accept_content=['pickle'],  # Ignore other content
     result_serializer='pickle',
 )
+'''
+import_single_post
+
+input: String (Path to json file)
+output: Void
+purpose: This function is used to import a single post from a json file to a database
+'''
+@app.task
+def import_single_post(path_to_post):
+    
+    # Read file
+    with open(path_to_post) as f:
+            data = json.load(f)
+
+    # Connect to database
+    db = utils.connect()
+
+    # filter fields to only include author, author_fullname, created_utc, full link, id, num_comments, score, selftext, title
+    post = {
+        'author': data[0]['data']['children'][0]['data']['author'],
```

Comment:
let's clean up these lines, e.g., 
 data[0]['data']['children'][0]['data']
seems to be the same for each

Reply:
Good catch, updated!